libass/ass.c

```diff
@@ -1520,19 +1520,43 @@ ASS_Track *ass_new_track(ASS_Library *library)
     return NULL;
 }
 
-int ass_track_set_feature(ASS_Track *track, ASS_Feature feature, int enable)
-{
-    switch (feature) {
-    case ASS_FEATURE_INCOMPATIBLE_EXTENSIONS:
-        //-fallthrough
+static const uint64_t supported_features =
 #ifdef USE_FRIBIDI_EX_API
-    case ASS_FEATURE_BIDI_BRACKETS:
-        track->parser_priv->bidi_brackets = !!enable;
+    1LL << ASS_FEATURE_BIDI_BRACKETS |
```

Comment:
Not an issue, but what libass style guide says about literal suffixes? Currently we have lowercase unsigned suffix (10u) and uppercase long long suffix (60LL). I think we should make that uniform. Personally I prefer uppercase hex digits and lowercase suffixes to not mix them.